metabase/metabase
# Fix embed loader not translated with #locale

Relate to #50182 

### Description

First [reported on Slack](https://metaboat.slack.com/archives/C063Q3F1HPF/p1733249842094189)

There were 2 problems.

1. The loading string is translated at load time, so it needs to be changed to a function
1. Since LocaleProvider is unblocking, we allow the loader to be rendered before the locale is loaded

I'm backporting this PR because the [feature targets 52](https://www.notion.so/metabase/Support-language-localization-in-static-embedding-0307665e3008435d9bb2b3fff8148d7d).

### How to verify

Create a public link of a question or a dashboard, and ensure they have `#locale=<your_target_locale>` in it. e.g. `#locale=ko` (Korean)

### Checklist
